instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -111,6 +113,16 @@ public OpenTelemetry openTelemetry(
       return autoConfiguredOpenTelemetrySdk.getOpenTelemetrySdk();
     }
 
+    /**
+     * Expose the {@link ConfigProperties} bean for use in other auto-configurations.
+     *
+     * <p>Why not use spring boot properties directly? <br>
+     * 1. issues with older spring boot versions <br>
```

Comment:
wouldn't these issues affect our (broader) usage of spring boot properties to configure the SDK?

Reply:
merging to unblock #11350, but i'd be in favor of removing this bullet (or explaining it better), since as written I think it creates more questions than it answers

Reply:
it's this style that silently returns false in boot 2

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11350/files#diff-74d5efe463708e7c420646f0b1a8b490c76528e0abba9e8605f6f79bc73e1c40L33

Reply:
> wouldn't these issues affect our (broader) usage of spring boot properties to configure the SDK?

no

Reply:
@zeitlinger 

> i'd be in favor of removing this bullet (or explaining it better), since as written I think it creates more questions than it answers

```suggestion
```

Reply:
(i'll send PR)